frontend/components/sidebar/left/SidebarLeft.vue

```diff
@@ -22,7 +22,7 @@
         '-mr-[0.55rem]': contentScrollable,
       }"
     >
-      <SearchBar class="mt-1" location="sidebar" />
+      <SearchBar class="mt-1" :location="DropdownLocation.SIDEBAR" />
```

Comment:
Checking on what the idea for this is, @tatianakharitonovich :) As it's `DropdownLocation`, is the thinking here for the options that would be being displayed for the search bar? If so, really good to think of this! Just let me know what the plan is here :)